var/spack/repos/builtin/packages/openblas/package.py

```diff
@@ -263,6 +263,9 @@ def make_defs(self):
             'FC={0}'.format(spack_fc),
         ]
 
+        if self.spec.os == 'monterey':
+            make_defs.append('MACOSX_DEPLOYMENT_TARGET=11.0')
```

Comment:
Same comment as https://github.com/BlueBrain/spack/pull/1489#discussion_r809082315 really

Reply:
Repushed without the change. And up-to-date with current `develop`.